index.ts

```diff
@@ -158,3 +158,6 @@ export * from "./src/NodeParser/VoidTypeNodeParser";
 export * from "./src/SchemaGenerator";
 
 export * from "./factory";
+
+import ts from "typescript";
+export { ts };
```

Comment:
Can we do

```suggestion
export { ts } from "typescript";
```

Reply:
I tried that initially, but got this error from compiler:
> 'ts' can only be imported by using 'import ts = require("typescript")' or a default import.ts(2616)